src/test/java/gov/loc/repository/bagit/verify/BagVerifierTest.java

```diff
@@ -194,9 +199,14 @@ public void testErrorWhenMissingPayloadDirectory() throws Exception{
   public void testErrorWhenMissingPayloadManifest() throws Exception{
     copyBagToTestFolder();
     Bag bag = reader.read(Paths.get(folder.getRoot().toURI()));
-    File manifestFile = new File(folder.getRoot(), "manifest-md5.txt");
-    manifestFile.delete();
-    
+    File manifestFile = new File(folder.getRoot(), "manifest-md5.txt");  
+    if (PathUtils.isWindows()){    	
+    	Path manifestPath = FileSystems.getDefault().getPath(manifestFile.getAbsolutePath());	
+    	Files.move(manifestPath, manifestPath.resolveSibling("renamedManifext"));
+    }
+    else {
+    	manifestFile.delete();
```

Comment:
It's the delete() of the temporary files that doesn't work on Windows -- the "under-the-hood" makes the file inaccessible, but never actually deletes it -- temp folders and files just don't get cleaned up in Windows, despite the "promise" of API -- 
Certainly we could move the *nix files and just do the same think for both --